src/components/AnnualReport/AGMDate.vue

```diff
@@ -1,68 +1,114 @@
 <template>
-  <v-card flat class="agm-date-container">
+  <v-card flat id="agm-date-container" v-if="isCoop">
     <div class="meta-container">
       <label>Annual General<br>Meeting Date</label>
 
-      <v-form ref="form" class="value date">
-        <v-menu
-          ref="menu"
-          v-model="menu"
-          :close-on-content-click="false"
-          :nudge-right="40"
-          transition="scale-transition"
-          offset-y
-          max-width="290"
-        >
-          <template v-slot:activator="{ on }">
-            <v-text-field
-              data-test-id="agm-date-text"
-              v-model="dateText"
-              :disabled="noAgm"
-              :rules="agmDateRules"
-              label="Annual General Meeting Date"
-              placeholder="Select your Annual General Meeting Date"
-              prepend-icon="mdi-calendar"
-              v-on="on"
-              readonly
-              filled
+      <div>
```

Comment:
This wrapper div was needed for the flex layout to work properly (ie, label on left, everything else on right).